### PR TITLE
Create a new function runTestsWithArgs

### PR DESCRIPTION
**Pull request by forki**
This will allow users to supply their own tests and still use the command line args

### CODE REVIEW

File: Expecto/Expecto.fs [hunk 1]
```diff
@@ -861,21 +861,27 @@ module Tests =
...
-      match testFromAssembly (Assembly.GetEntryAssembly()) with
-      | Some t -> t
-      | None -> TestList ([], Normal)
+  /// Runs all given tests with the supplied command-line options. Returns 0 if all tests passed, otherwise 1
+  let runTestsWithArgs config args tests =
     let config, isList = args |> ExpectoConfig.fillFromArgs config
     let tests = tests |> config.filter
-    if isList then listTests tests; 0 else runTests config tests
+    if isList then 
+        listTests tests
```
Comment (haf) [inline Expecto/Expecto.fs]: 2 space indentation.

File: Expecto/Expecto.fs [hunk 2]
```diff
@@ -861,21 +861,27 @@ module Tests =
...
-  /// Runs tests in this assembly with supplied command-line options. Returns 0 if all tests passed, otherwise 1
-  let runTestsInAssembly config args =
-    let tests =
-      match testFromAssembly (Assembly.GetEntryAssembly()) with
-      | Some t -> t
-      | None -> TestList ([], Normal)
+  /// Runs all given tests with the supplied command-line options. Returns 0 if all tests passed, otherwise 1
+  let runTestsWithArgs config args tests =
     let config, isList = args |> ExpectoConfig.fillFromArgs config
     let tests = tests |> config.filter
```
Comment (haf) [inline Expecto/Expecto.fs]: When you're here changing the code, please make single-param functions be called without `|>`

File: Expecto/Expecto.fs [hunk 3]
```diff
@@ -861,21 +861,27 @@ module Tests =
...
           getLogger = fun name -> LiterateConsoleTarget(name, config.verbosity) :> Logger }
     run config.printer tests
 
-  /// Runs tests in this assembly with supplied command-line options. Returns 0 if all tests passed, otherwise 1
-  let runTestsInAssembly config args =
-    let tests =
-      match testFromAssembly (Assembly.GetEntryAssembly()) with
-      | Some t -> t
-      | None -> TestList ([], Normal)
+  /// Runs all given tests with the supplied command-line options. Returns 0 if all tests passed, otherwise 1
```
Comment (haf) [inline Expecto/Expecto.fs]: 80 chars

File: Expecto/Expecto.fs (hunk 2 above)
Comment (forki) [inline Expecto/Expecto.fs]: that was just moved from other place. but ok will remove the pipe

File: Expecto/Expecto.fs (hunk 3 above)
Comment (forki) [inline Expecto/Expecto.fs]: fixed

**Comment by forki** [comment]
rebased after recent merge of conflicting PR

**Comment by haf** [comment]
Will need a rebase again, I'm afraid ;)

**Comment by forki** [comment]
done

**Comment by haf** [comment]
Super! Thanks!

**Comment by forki** [comment]
cool. thanks for merging. 
would be very happy if you could release. Then I could add the thingy to the fake helper and then my real world project would fail on CI and I could blame my coworkers ;-)

**Comment by haf** [comment]
LOL

**Comment by haf** [comment]
Ok, could you please write a paragraph of release notes?

**Comment by haf** [comment]
Also, please – docs PR into the README.

**Comment by forki** [comment]
see https://github.com/haf/expecto/pull/50

**Pull request merged**